samples/cpp/object_detection_sample_ssd/main.cpp

```diff
@@ -326,16 +304,12 @@ int main(int argc, char* argv[]) {
                 }
             }
         }
-        // -----------------------------------------------------------------------------------------------------
 
-        // --------------------------- Step 7. Do inference
-        // ---------------------------------------------------------
+        // Step 7. Do inference
```

Comment:
comments should be unified in all samples (see https://github.com/openvinotoolkit/openvino/blob/5b2a78be78b4aa0d391cb1f1b5d85ec5daec11fa/samples/cpp/hello_classification/main.cpp#L50)

Please, stick to a common approach either with `--------` or w/o them - up to you.